app/controllers/events/list.js

```diff
@@ -38,6 +38,15 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         width           : 180,
         cellComponent   : 'ui-table/cell/cell-roles'
       },
+      {
+        name            : this.l10n.t('Tickets'),
+        width           : 130,
+        valuePath       : 'totalsales',
+        extraValuePaths : ['tickets'],
+        isSortable      : true,
```

Comment:
I think there is no need to make this column osrtable and if you are making it then for which parameter u r sorting it? and also sorting is not working here

Reply:
Sir I am trying to sort it on the bases of sum of total ticket sold for each particular event.
Yes I know sir , I am working on sorting part but couldn't find a way through. Can you help me on it Sir ?

Reply:
I  think you can't sort on the basis of the sum of the total ticket, it is a computed property not a attribute.

Reply:
ok sir.

Reply:
@maze-runnar sir I tried declaring an attribute **ticketsold** in event model and given it the value of sum of total ticket sold for each particular event, and also set it as valuePath but still sorting is not working for it ?

Reply:
> I think there is no need to make this column sortable and if you are making it then for which parameter u r sorting it? and also sorting is not working here

Sir I am not able to solve sorting issue.